Cargo.toml

```diff
@@ -30,11 +19,12 @@ chrono = "0.4.31"
 clap = "4.4.6"
 foreign-types-shared = "0.3.1"
 futures = "0.3.28"
-histogram = { workspace = true }
+histogram = { version = "0.9.0", features = ["serde"] }
```

Comment:
Is this still needed as a direct dependency?

Reply:
Good catch, thanks.